releases/v1.1.4.md

```diff
@@ -0,0 +1,80 @@
+- The [`IMPORT`](../v1.1/import.html) statement now supports `201` and `204` return codes from HTTP storage. [#20516](https://github.com/cockroachdb/cockroach/pull/20516)
+
+### Command-Line Changes
+
+- Client [`cockroach` commands](../v1.1/cockroach-commands.html) that use SQL (`cockroach sql`, `cockroach node ls`, etc.) now print a warning if the server is running an older version of CockroachDB than the client. Also, this and other warning messages are now clearly indicated with the "warning:" prefix. [#20945](https://github.com/cockroachdb/cockroach/pull/20945)
+- The [`cockroach sql`](../v1.1/use-the-built-in-sql-client.html) command no longer fails when the server is running a version of CockroachDB that does not support the `sql_safe_updates` [session variable](../v1.1/set-vars.html#supported-variables). [#20945](https://github.com/cockroachdb/cockroach/pull/20945)
+- The [`cockroach sql`](../v1.1/use-the-built-in-sql-client.html) command now performs client-side syntax checking automatically only if the client and server are running the same version of CockroachDB. This restriction will be lifted in CockroachDB v2.0. [#21135](https://github.com/cockroachdb/cockroach/pull/21135)
+- Reduced temporary disk space usage for the `debug compact` command. [#20502](https://github.com/cockroachdb/cockroach/pull/20502)
+
+### Bug Fixes
+
+- Improved rebalancing to make thrashing back and forth between nodes much less likely, including when [localities](../v1.1/start-a-node.html#locality) have very different numbers of nodes. [#20934](https://github.com/cockroachdb/cockroach/pull/20934)
+- Fixed a scenario where a range that is too big to snapshot can lose availability even with a majority of nodes alive. [#20906](https://github.com/cockroachdb/cockroach/pull/20906)
+- Fixed a replica corruption that could occur if a process crashed in the middle of a range split. [#20939](https://github.com/cockroachdb/cockroach/pull/20939)
+- Fixed an issue where seemingly irrelevant error messages were being returned for certain [`INSERT`](../v1.1/insert.html) statements.[#20855](https://github.com/cockroachdb/cockroach/pull/20855)
+- Crash details are now properly copied to the log file even when a node was started with `--logtostderr` as well as in other circumstances when crash details could be lost previously. [#20842](https://github.com/cockroachdb/cockroach/pull/20842)
+- Fixed an issue where wildcards (`_`, `%`) and `\` escaped characters in `LIKE` patterns were not handled properly. [#20654](https://github.com/cockroachdb/cockroach/pull/20654)
+- Fixed the return type signature of the `array_positions()` [built-in function](../v1.1/functions-and-operators.html). This caused a crash if the function was used in query execution by the distributed execution engine. [#20652](https://github.com/cockroachdb/cockroach/pull/20652)
+- Fix a crash caused by null [collated strings](../v1.1/collate.html). [#20648](https://github.com/cockroachdb/cockroach/pull/20648)
```

Comment:
nit: 'Fix' to 'Fixed' to maintain consistency

Reply:
Thanks, @Amruta-Ranade. I'm going to let TC finish so I can fix any broken links with this update.

Reply:
Done.